react/next/Styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
```

Comment:
why styled components if we can use css modules?

Reply:
Removed it